R/calc_AUC.R

```diff
@@ -0,0 +1,99 @@
+require(caTools)
+
+calc_AUC <-
+  function(qp,
+           test_matrix_Xi,
+           test_matrix_Xip,
+           test_yi) {
+    rankdiff <- function(qp, matrix_Xi, matrix_Xip, t) {
```

Comment:
I would suggest to change the name of the `t` argument, to avoid conflict/confusion with `base::t`